### PR TITLE
added custom port option for rocketchat

### PR DESCRIPTION
**Pull request by atighineanu**
Closes: https://github.com/containrrr/shoutrrr/issues/76
Closes: https://github.com/containrrr/shoutrrr/issues/71
Added custom port option for --url rocketchat://Host:Port/....

### CODE REVIEW

File: pkg/services/rocketchat/rocketchat.go [hunk 1]
```diff
@@ -30,18 +29,28 @@ func (service *Service) Initialize(configURL *url.URL, logger *log.Logger) error
...
+	}
 	if res.StatusCode != http.StatusOK {
 		return fmt.Errorf("failed to send notification to service, response status code %s", res.Status)
 	}
 	return err
 }
 
 func buildURL(config *Config) string {
-	return fmt.Sprintf("https://%s/hooks/%s/%s", config.Host, config.TokenA, config.TokenB)
+	if config.Port != "443" && config.Port != "" {
```
Comment (simskij) [inline pkg/services/rocketchat/rocketchat.go:49]:
```suggestion
	if config.Port != "" {
```

No need to do an explicit check for 443. It shouldn't matter if it gets appended or not given that `https` is 443 anyway.

**Review by simskij — CHANGES_REQUESTED** [review]
Could you please add some tests for this as well? One where Port is set, and one where it's not.

File: pkg/services/rocketchat/rocketchat.go (hunk 1 above)
Comment (atighineanu) [reply pkg/services/rocketchat/rocketchat.go]: DONE.

**Review by simskij — APPROVED** [review]
lgtm. thanks again for contributing!

**Pull request merged**